<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scancrop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen_template"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_100"/>
    <addaction name="actionZoom_25"/>
    <addaction name="actionZoom_to_Fit"/>
    <addaction name="actionZoom_Width"/>
    <addaction name="actionToggle_image_visible"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionPan"/>
    <addaction name="actionBox"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect_All"/>
    <addaction name="actionFind_images"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen_template"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_100"/>
   <addaction name="actionZoom_25"/>
   <addaction name="actionZoom_to_Fit"/>
   <addaction name="actionZoom_Width"/>
   <addaction name="separator"/>
   <addaction name="actionRotate_CounterClockwise"/>
   <addaction name="actionRotate_Clockwise"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionPan"/>
   <addaction name="actionBox"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="topLeftView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="topRightView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="bottomLeftView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="bottomRightView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save cropped image(s)</string>
   </property>
   <property name="toolTip">
    <string>Save cropped image(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionZoom_to_Fit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom_fit.png</normaloff>:/images/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to Fit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoom_Width">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom_width.png</normaloff>:/images/zoom_width.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Width</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/tool-pan.png</normaloff>:/images/tool-pan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/cursor.png</normaloff>:/images/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/draw_box.png</normaloff>:/images/draw_box.png</iconset>
   </property>
   <property name="text">
    <string>Box</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionRotate_Clockwise">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/shape_rotate_clockwise.png</normaloff>:/images/shape_rotate_clockwise.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Clockwise</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionRotate_CounterClockwise">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/shape_rotate_anticlockwise.png</normaloff>:/images/shape_rotate_anticlockwise.png</iconset>
   </property>
   <property name="text">
    <string>Rotate CounterClockwise</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionZoom_100">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom100.png</normaloff>:/images/zoom100.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSave_template">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/save_template.png</normaloff>:/images/save_template.png</iconset>
   </property>
   <property name="text">
    <string>Save template</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen_template">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open_template.png</normaloff>:/images/open_template.png</iconset>
   </property>
   <property name="text">
    <string>Open template...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoom_25">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom25.png</normaloff>:/images/zoom25.png</iconset>
   </property>
   <property name="text">
    <string>Zoom 25%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/book_previous.png</normaloff>:/images/book_previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="toolTip">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/book_next.png</normaloff>:/images/book_next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="toolTip">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionFind_images">
   <property name="text">
    <string>Detect image edges</string>
   </property>
   <property name="toolTip">
    <string>Find photos on the page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionToggle_image_visible">
   <property name="text">
    <string>Toggle image visible</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
